python/ray/gateway.py

```diff
@@ -0,0 +1,110 @@
+import click
+from flask import Flask, request, send_file
+import io
+import pickle
+import pyarrow
+from pyarrow import plasma as plasma
+import sys
+
+app = Flask(__name__)
```

Comment:
Flask was the "obvious" choice for just ramming through since it's a pretty simple, widely used web application framework. It might be useful for serving up dashboards to the user, but I wouldn't use it for data / code transfer in anything except early prototypes. For data / code, Flask has way too many other things we don't need (e.g., URL routing) and it's not even clear that REST is the right thing to do (besides the fact it's simple).

Reply:
Ok sounds good.